docs/paper/paper.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'SWMManywhere: Synthesise Urban Drainage Network Models Anywhere in the World'
+tags:
+  - python
+  - stormwater
+  - hydrology-stormwater-analysis
+  - swmm5
+  - swmm
+  - hydraulic-modelling
+authors:
+  - family-names: "Dobson"
+    given-names: "Barnaby"
+    orcid: "https://orcid.org/0000-0002-0149-4124"
+    affiliation: 1
+  - family-names: "Alonso-Álvarez"
+    given-names: "Diego"
+    orcid: "https://orcid.org/0000-0002-0060-9495"
+    affiliation: 1
+  - family-names: "Chegini"
+    given-names: "Taher"
+    orcid: "https://orcid.org/0000-0002-5430-6000"
+    affiliation: 2
+affiliations:
+ - name: Imperial College London, UK
+   index: 1
+ - name: Purdue University, US
+   index: 2
+date: 16 October 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Urban drainage network models (UDMs) are useful for a variety of purposes, most notably simulating and alleviating pluvial flooding. Despite the clear usefulness of UDMs, they are often not used owing to the difficulty and expense of creating them. This creates a significant gap for users attempting to generate UDMs if they are not able to perform an expensive underground survey. A variety of literature exists to derive such UDMs from GIS data, producing hydraulically feasible models that closely approximate real-world systems [@Blumensaat2012-hd;@Chahinian2019-lg;@Reyes-Silva2022-pr]. We identify some key limitations of these approaches, most notably the lack of automatic data acquisition and preprocessing, that all approaches are closed-source to date, and that a key feature of such an approach should be to facilitate extension and customisation.
+
+# Statement of need
+
+`SWMManywhere` is an open-source Python package designed for the global synthesis of urban drainage networks. SWMManywhere integrates publicly available geospatial data and automates data acquisition and preprocessing, reducing the technical burden on users. The CLI offers a flexible workflow, providing an accessible entry point to using and customising synthesis. Its parameterized design enables detailed sensitivity analyses, allowing users to understand and manage uncertainties inherent in urban drainage modelling [@Dobson2024-dv]. By emphasizing user control, SWMManywhere allows tuning of outputs with parameters to meet local requirements, making it adaptable to a wide range of scenarios. Designed for both researchers and practitioners in urban water management, SWMManywhere responds to the limitations of existing methods by providing an end-to-end, open-source, and customisable solution.
+
+## Outlook
```

Comment:
It feels odd that most of the paper talks about what it does not do, yet, rather than what it does, its unique features or what does better than other approaches. I know you do need to talk about this - and you phrased it very well as research opportunities rather than software limitations - but even so.

I would include a section on Features - or maybe as part of the statement of need - and a picture (maybe a map of the notebook with the calculated swage system) and reduce this section a little bit..

Reply:
good suggestions!